server/src/main/java/org/elasticsearch/common/lucene/Lucene.java

```diff
@@ -1017,20 +1017,18 @@ public CacheHelper getReaderCacheHelper() {
     }
 
     /**
-     * Scans sequence numbers (i.e., {@link SeqNoFieldMapper#NAME}) between {@code fromSeqNo}(inclusive) and {@code toSeqNo}(inclusive)
-     * in the provided directory reader. This method invokes the callback {@code onNewSeqNo} whenever a sequence number value is found.
+     * Scans sequence numbers (i.e., {@link SeqNoFieldMapper#NAME}) from {@code fromSeqNo}(inclusive) in the provided directory reader.
+     * This method invokes the callback {@code onNewSeqNo} whenever a sequence number value is found.
      *
      * @param directoryReader the directory reader to scan
      * @param fromSeqNo       the lower bound of a range of seq_no to scan (inclusive)
-     * @param toSeqNo         the upper bound of a range of seq_no to scan (inclusive)
      * @param onNewSeqNo      the callback to be called whenever a new valid sequence number is found
      */
-    public static void scanSeqNosInReader(DirectoryReader directoryReader, long fromSeqNo, long toSeqNo,
-                                          LongConsumer onNewSeqNo) throws IOException {
+    public static void scanSeqNosInReader(DirectoryReader directoryReader, long fromSeqNo, LongConsumer onNewSeqNo) throws IOException {
```

Comment:
I would prefer to keep this method signature with fromSeqNo and toSeqNo, and instead invoke it with Long.MAX_VALUE when we have a commit from < 6.6.2